### PR TITLE
OSDOCS-11019: updates command parameters under installing in MicroShift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.14, 4.15, 4.16, 4.17

Issue:
[OSDOCS-11019](https://issues.redhat.com/browse/OSDOCS-11019)

Link to docs preview:
[Creating the RHEL for Edge image - edge image](https://78516--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree.html#microshift-creating-ostree-iso_microshift-embed-in-rpm-ostree)
[Creating the RHEL for Edge image - offline use](https://78516--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree-offline-use.html#microshift-creating-ostree-iso_microshift-embed-rpm-ostree-offline-use)

QE review:
- [x] QE has approved this change.
- [x] SME has approved this change.

### CODE REVIEW

**Comment by aravipra** [comment]
@DanielFroehlich I see that line 92 has "minimal-microshift-server" mentioned should this be replaced as well? Also please confirm the versions to which the changes is to be applied.

**Review by DanielFroehlich — APPROVED** [review]
/LGTM

**Comment by DanielFroehlich** [comment]
> @DanielFroehlich I see that line 92 has "minimal-microshift-server" mentioned should this be replaced as well? Also please confirm the versions to which the changes is to be applied.

No, line 92 is something different - that is not the blueprint name, but the name of the runtime container.
Backport down to 4.14 would be nice if not too much of a hassle.

**Review by gwynnemonahan — COMMENTED** [review]
Hey @aravipra,

LGTM! Thanks!

File: modules/microshift-creating-ostree-iso.adoc
Comment (skopacz1) [inline modules/microshift-creating-ostree-iso.adoc:26]: Per the "Important" note in this section of the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values), we shouldn't italicize user replaced values. Apparently this is an exception to the SSG, so apologies for the inconvenience if you were following that as a model:
```suggestion
$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" <microshift_blueprint> edge-container | awk '{print $2}')<1>
----
<1> Replace `<microshift_blueprint>` with the name of your blueprint.
```

**Review by skopacz1 — COMMENTED** [review]
Just one nit, let me know when it's addressed and I can proceed to merge!

File: modules/microshift-creating-ostree-iso.adoc
Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc:26]: Hi Sebastian,
In line 26 , using back ticks for user values is not followed.
in line 24, I do agree with the suggestion made but I would retain the changes as is to maintain consistency in the doc. I can take this suggestion in another jira and make the suggested changes to all the places in the doc and remove the italics in command.

For now I think this change is out of scope for this PR.

Also, I have made edits to line 39, 42, 44. Please review before merging this PR.

Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc:26]: @DanielFroehlich and @dhensel-rh Please review the lines 39,42 and 44. Changes are done for the blueprint name.

Comment (skopacz1) [reply modules/microshift-creating-ostree-iso.adoc:26]: Hi Ashwini, unless I'm mistaken, there is no place in our docs where we use italics for user-replaced values, or if there is, they incorrect and should be fixed. This is the OCP docs guidelines section on user-replaced values in command, which has an Important admonition explicitly stating to not use italics around these values: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values

Any rule in that guideline takes priority over the SSG, since that guideline is more specific to our product docs. The italics must be removed in both lines 24 and 26.

Additionally, please take a look at this section of our guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

In that table, you'll find a row for "System or software variable to be replaced by the user", which refers to user replaced values, and you'll see that they are wrapped in backticks. This means that whenever user-replaced values are used outside of command syntax, like the callout label on line 26 here, they must have backticks.

Please let me know when you've implemented the feedback in my original suggestion.

File: modules/microshift-creating-ostree-iso.adoc
Comment (skopacz1) [inline modules/microshift-creating-ostree-iso.adoc:44]: I notice that you've also made these changes after my original merge review. In this case, `minimal-microshift` should stay in the terminal block, because this block is an example output of the `sudo composer-cli compose status` command. 

Because this is an example, it should have a specific example value instead of a generic user-replaced value. User-replaced values are reserved for instances where users must fill in their own information in a command or file in order to perform a procedure. In this case, this is just an example, so mentioning a specific value is appropriate here, even if a user might see different output on their own cluster.

Please revert the changes in these lines.

**Review by skopacz1 — COMMENTED** [review]
Hi Ashwini, I left a few more comments, please let me know when they are addressed and I can proceed to merge.

**Comment by skopacz1** [comment]
It's the end of my review week and it doesn't seem like I'll have a chance to merge by EOD, so I'll drop the merge review labels. Please put the merge review needed label up again when this is ready for another review. Thanks!

File: modules/microshift-creating-ostree-iso.adoc
Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc:26]: As discussed, OCP has made the exception to this SSG rule because it was too much effort to update that repo with the italics. Microshift is a new docset and follows the SSG. So, this guideline can be ignored and the italics can be retained to maintain consistency.
cc @ShaunaDiaz

**Comment by aravipra** [comment]
> Hi Ashwini, I left a few more comments, please let me know when they are addressed and I can proceed to merge.

same response as mentioned above.
As discussed, OCP has made the exception to this SSG rule because it was too much effort to update that repo with the italics. Microshift is a new docset and follows the SSG. So, this guideline can be ignored and the italics can be retained to maintain consistency.

File: modules/microshift-creating-ostree-iso.adoc
Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc:44]: for lines 24 and 26:
same response as mentioned earlier which is:
As discussed, OCP has made the exception to this SSG rule because it was too much effort to update that repo with the italics. Microshift is a new docset and follows the SSG. So, this guideline can be ignored and the italics can be retained to maintain consistency.

Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc]: changes are made for the output file in lines 42 and 50

**Review by ShaunaDiaz — COMMENTED** [review]
please update line 91/92 to show microshift-blueprint-server since the name of the blueprint file changed; see https://github.com/openshift/openshift-docs/pull/78516/files#diff-49a96ddb162ad0ffa7477c5f65bacec18555f03b0ff2a5de5692a89b2e9b9471L91

**Comment by aravipra** [comment]
> please update line 91/92 to show microshift-blueprint-server since the name of the blueprint file changed; see https://github.com/openshift/openshift-docs/pull/78516/files#diff-49a96ddb162ad0ffa7477c5f65bacec18555f03b0ff2a5de5692a89b2e9b9471L91

@shauna, I had asked this to daniel. He had informed me not to change this. 
https://github.com/openshift/openshift-docs/pull/78516#issuecomment-2216724095

File: modules/microshift-creating-ostree-iso.adoc
Comment (jeana-redhat) [inline modules/microshift-creating-ostree-iso.adoc:24]: Need a space before the callout number
```suggestion
$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/{op-system-version-major}/$(uname -m)/edge" __<microshift_blueprint>__ edge-container | awk '{print $2}') <1>
```

File: modules/microshift-creating-ostree-iso.adoc
Comment (jeana-redhat) [reply modules/microshift-creating-ostree-iso.adoc:26]: There are two issues here: the italics in the command block (which based on discussion above I understand is a difference between OCP and MicroShift) and the backticks around the value in the callout explanation. 

The latter item seems to also be contrary to [the SSG](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values):
"If the user-replaced value is referencing a value in code or in a command that is normally monospace, also use monospace for the user-replaced value"

If I am reading right, it sounds like this is consistently not used in the existing docs.

So, if i am following, what makes sense to do here is:
1. leave the italics alone, MicroShift uses them and that's fine
2. the omitted backticks can be tackled across the docs as a separate effort.

Is that right? Just making sure I have understood - fwiw if that is what you are proposing, I think that sounds reasonable :)

**Review by jeana-redhat — COMMENTED** [review]
One tiny typo, and one question to make sure I've understood the resolution of your discussion earlier. Otherwise LGTM!

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/microshift-creating-ostree-iso.adoc
Comment (ShaunaDiaz) [reply modules/microshift-creating-ostree-iso.adoc:26]: It does.

Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc:26]: Yes that is right @jeana-redhat

File: modules/microshift-creating-ostree-iso.adoc
Comment (aravipra) [reply modules/microshift-creating-ostree-iso.adoc]: done

File: modules/microshift-creating-ostree-iso.adoc
Comment (jeana-redhat) [reply modules/microshift-creating-ostree-iso.adoc]: Thank you for confirming all that! SGTM.

**Pull request merged**
